helm_deploy/cla-frontend/values.yaml

```diff
@@ -130,3 +130,7 @@ envVars:
     secret:
       name: zendesk-api
       key: username
+  FAMILY_ISSUE_FEATURE_FLAG:
+    configmap:
+      name: family-issue
+      key: family-issue-on
```

Comment:
The `ConfigMap` doesn't exist on training or production namespaces yet.
As this configmap lookup has not been set to optional, the deployment will fail (as you saw previously on other environments)

Is your plan to create those configmaps before this is merged and deployed, or did you intend it to be optional?

Reply:
I'll create them before this is merged

Reply:
OK. Let me know when you have, and I'll be able to approve the PR

Reply:
I've added them in now